src/util/dev/server.ts

```diff
@@ -239,6 +239,34 @@ export default class DevServer {
         }
       }
     }
+
+    // Special case `@now/static-build`,
+    // since it won't trigger builds otherwise
+    ((nowConfig && nowConfig.builds) || []).map((build) => {
```

Comment:
```suggestion
    ((nowConfig && nowConfig.builds) || []).forEach((build) => {
```

It looks like the returned value from map is not used so probably use a for each instead